diagnostic.md

```diff
@@ -92,7 +119,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+color: yellow;
+font-family: sans-serif;
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive